src/__tests__/__snapshots__/Request.test.js.snap

```diff
@@ -1,202 +1,16 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Request should render with loaded props 1`] = `
-<SceneWrapper
-  background="header"
-  hasTabs={false}
->
-  <View
-    style={
-      Object {
-        "flex": 1,
-        "justifyContent": "flex-start",
-        "paddingHorizontal": 22,
-      }
+<ActivityIndicator
```

Comment:
Something has gone wrong with the unit tests. We are rendering a loading indicator, instead of the actual scene we are testing.